grpc.kafka.echo/chart/templates/service-kafka.yaml

```diff
@@ -20,7 +20,7 @@ spec:
             - /bin/sh
             - -c
             - /opt/bitnami/scripts/kafka/setup.sh && kafka-storage.sh format --config "${KAFKA_CONF_FILE}" --cluster-id "lkorDA4qT6W1K_dk0LHvtg" --ignore-formatted && /opt/bitnami/scripts/kafka/run.sh
-          image: "bitnami/kafka:3.1.0"
+          image: "bitnami/kafka:3.5"
```

Comment:
We've run into some issues by updated to the later releases of `kafka` so it might be a good idea to manage this version upgrade independent of these example folders changes.

For example, there is also a `3.5.1` and a `3.6` release of Apache Kafka and corresponding Bitnami docker images for us to choose from after checking for any issues reported on the various releases.

Reply:
We were on `3.2` before the conference when we found the issue and went back to `3.1.0`. Akram said the issue was fixed. [3.5](https://hub.docker.com/r/bitnami/kafka/tags?page=1&name=3.5) is the latest tag for that minor version and will pull any new patch versions should they become available. I didn't want to use `3.6` since they haven't made a first patch to it yet. I would just like to move to `3.2+` since they have support for arm OS.